.pipelines/stages/jobs/py-validation-job.yml

```diff
@@ -220,14 +222,18 @@ jobs:
           python -m pip install -r test/python/directml/torch/requirements.txt
           python -m pip install -r test/python/directml/ort/requirements.txt
         }
+        elseif ("$(arch)" -eq "arm64") {
+          python -m pip install numpy<2
```

Comment:
Which package has conflict with numpy2?

Reply:
Not sure but the other eps use numpy<2 in requirements.txt